lmn/tests/test_views.py

```diff
@@ -469,6 +469,24 @@ def test_correct_templates_uses_for_notes(self):
         self.assertTemplateUsed(response, 'lmn/notes/new_note.html')
 
 
+class TestUserNotes(TestCase):
+
+    fixtures = [ 'testing_users', 'testing_artists', 'testing_venues', 'testing_shows', 'testing_notes' ]
+
+    def test_delete_own_note(self):
+        self.client.force_login(User.objects.first())
+        response = self.client.post(reverse('delete_note', args=(1,)), follow=True)
+        note_2 = Note.objects.filter(pk=1).first()
```

Comment:
Should change name of variable note_2 to just note if using pk=1? I think note_2 would represent a pk=2.

Reply:
Good point, I just changed it. Thanks!